runtime/caml/custom.h

```diff
@@ -51,7 +51,7 @@ struct custom_operations {
 extern "C" {
 #endif
 
-CAMLextern uintnat caml_custom_major_ratio;
+CAMLextern _Atomic uintnat caml_custom_major_ratio;
```

Comment:
This should be `atomic_uintnat` for C++ compatibility (C++ doesn't understand `_Atomic`). The test `cxx-api/all_includes.ml` should have caught this error. I'll investigate why it didn't.